cypress/integration/data-import/match-on-location.spec.js

```diff
@@ -0,0 +1,362 @@
+import getRandomPostfix from '../../support/utils/stringTools';
+import TestTypes from '../../support/dictionary/testTypes';
+import DevTeams from '../../support/dictionary/devTeams';
+import NewMatchProfile from '../../support/fragments/data_import/match_profiles/newMatchProfile';
+import NewActionProfile from '../../support/fragments/data_import/action_profiles/newActionProfile';
+import NewJobProfile from '../../support/fragments/data_import/job_profiles/newJobProfile';
+import SettingsMenu from '../../support/fragments/settingsMenu';
+import MatchProfiles from '../../support/fragments/data_import/match_profiles/matchProfiles';
+import FieldMappingProfiles from '../../support/fragments/data_import/mapping_profiles/fieldMappingProfiles';
+import ActionProfiles from '../../support/fragments/data_import/action_profiles/actionProfiles';
+import JobProfiles from '../../support/fragments/data_import/job_profiles/jobProfiles';
+import TopMenu from '../../support/fragments/topMenu';
+import DataImport from '../../support/fragments/data_import/dataImport';
+import FileDetails from '../../support/fragments/data_import/logs/fileDetails';
+import Logs from '../../support/fragments/data_import/logs/logs';
+import DataImportSettingsJobProfiles from '../../support/fragments/settings/dataImport/dataImportSettingsJobProfiles';
+import SearchInventory from '../../support/fragments/data_import/searchInventory';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
```

Comment:
'InventoryInstance' is defined but never used. Allowed unused vars must match /React/u

Reply:
Done